client/pruning/main.go

```diff
@@ -21,7 +21,7 @@ const FlagAppDBBackend = "app-db-backend"
 
 // PruningCmd prunes the sdk root multi store history versions based on the pruning options
```

Comment:
That's breaking, can you add a changelog (or revert)?
Moreover, if you don't revert, could you update the comment above?

Reply:
Sure!  so, in fact this is what revive does.